hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2935,7 +2935,7 @@
     <description>
       The base dir for HTTP Jetty server to extract contents. If this property
       is not configured, by default, Jetty will create a directory inside the
-      directory named by the java.io.tmpdir System property（/tmp by default）.
+      directory named by the ozone.metadata.dirs（/tmp/metadata)
```

Comment:
@ArafatKhan2198 , thanks for working on this patch, however it seems not correct as /tmp is anyway being cleaned up, I think this description is incorrect because "ozone.metadata.dirs" will vary for each component like SCM, Datanode, OM, Recon etc for their respective node and it will not be /tmp as HttpServer also will be started as separate process on each of the Ozone component.

Reply:
Thanks for the suggestion. I'll make the changes to the description.